src/documents/Enactors.md

```diff
@@ -25,16 +25,16 @@ Each of these is explained below.
 Enactors must be defined using the `fluid.prefs.enactor` [grade](ComponentGrades.md), as shown in the following code block:
 ```javascript
 fluid.defaults("my.pref.enactor", {
-    gradeNames: ["fluid.prefs.enactor", "autoInit"],
+    gradeNames: ["fluid.prefs.enactor"],
     ...
 });
 ```
 
 Enactors are, by default, [standard relay components](ComponentGrades.md
```

Comment:
Lingering reference to "standard relay components." Should be "model components."